src/checkout-buttons/strategies/googlepay/googlepay-button-strategy.ts

```diff
@@ -68,11 +67,13 @@ export default class GooglePayButtonStrategy implements CheckoutButtonStrategy {
     @bind
     private async _handleWalletButtonClick(event: Event): Promise<void> {
         event.preventDefault();
+        const cart = this._store.getState().cart.getCartOrThrow();
+        const hasPhysicalItems = cart.lineItems.physicalItems.length !== 0;
```

Comment:
We've just recently added [getShippableItemsCount](https://github.com/bigcommerce/checkout-sdk-js/blob/42761dce0f08b5a70fa874c1d176aeaa185f38a4/src/shipping/getShippableItemsCount.ts#L5), maybe you can use it here. 💅

Reply:
thanks @vmparra that makes things easier.